circleci: failed tests (15170): TestRaftReplayProtectionInTxn TestScannerEmptyRangeSet

The following test appears to have failed:

[#15170](https://circleci.com/gh/cockroachdb/cockroach/15170):

```
I160324 17:50:32.918878 storage/replica.go:1345  reproposing 1 commands after empty entry
I160324 17:50:32.919858 storage/replica_command.go:517  auto-gc'ed aa8d9a17 (1 intents)
E160324 17:50:32.921450 storage/replica.go:1429  [node=1,store=1,range=1] error executing raft command: write at timestamp 0.000000000,14 too old; wrote at 0.000000000,15
I160324 17:50:32.921924 storage/replica.go:1642  found sequence cache entry for aa8d9a17@epo=0,seq=1: epo=0,seq=1
I160324 17:50:32.922784 storage/engine/rocksdb.go:149  closing in-memory rocksdb instance
--- FAIL: TestRaftReplayProtectionInTxn (0.08s)
	replica_test.go:2195: expected WriteTooOldError; got retry txn "test" id=aa8d9a17 key="a" rw=false pri=0.02049753 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,14 orig=0.000000000,14 max=0.000000000,14 wto=false
=== RUN   TestReplayProtection
W160324 17:50:32.994488 gossip/gossip.go:898  not connected to cluster; use --join to specify a connected node
I160324 17:50:32.997450 raft/raft.go:459  [group 1] 1 became follower at term 5
I160324 17:50:32.997654 raft/raft.go:235  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I160324 17:50:32.997775 raft/raft.go:539  [group 1] 1 is starting a new election at term 5
I160324 17:50:32.997894 raft/raft.go:472  [group 1] 1 became candidate at term 6
I160324 17:50:32.997978 raft/raft.go:521  [group 1] 1 received vote from 1 at term 6
I160324 17:50:32.998144 raft/raft.go:500  [group 1] 1 became leader at term 6
I160324 17:50:32.999649 storage/replica.go:1833  [node=1,store=1,range=1] gossiping cluster id "75c7b350-5ed2-4731-964c-9148be7d1204" from store 1, range 1
--
I160324 17:50:35.008869 storage/scanner_test.go:257  1: average scan: 25.346581ms
--- PASS: TestScannerTiming (0.22s)
=== RUN   TestScannerPaceInterval
--- PASS: TestScannerPaceInterval (0.01s)
=== RUN   TestScannerEmptyRangeSet
--- FAIL: TestScannerEmptyRangeSet (0.01s)
	scanner_test.go:317: expected three loops; got 4
=== RUN   TestSequenceCachePutGetClearData
I160324 17:50:35.038633 storage/engine/rocksdb.go:149  closing in-memory rocksdb instance
--- PASS: TestSequenceCachePutGetClearData (0.01s)
=== RUN   TestSequenceCacheEmptyParams
I160324 17:50:35.045564 storage/engine/rocksdb.go:149  closing in-memory rocksdb instance
--- PASS: TestSequenceCacheEmptyParams (0.01s)
=== RUN   TestSequenceCacheCopyInto
I160324 17:50:35.058326 storage/engine/rocksdb.go:149  closing in-memory rocksdb instance
I160324 17:50:35.058463 storage/engine/rocksdb.go:149  closing in-memory rocksdb instance
--
I160324 17:51:14.572340 storage/raft_transport.go:222  remote node 2 at 127.0.0.1:45654 closed Raft transport with error: rpc error: code = 13 desc = "transport is closing"
--- PASS: TestInOrderDelivery (0.74s)
=== RUN   Example_rebalancing
W160324 17:51:14.575004 gossip/gossip.go:898  not connected to cluster; use --join to specify a connected node
--- PASS: Example_rebalancing (0.93s)
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	45.414s
=== RUN   TestBatchBasics
I160324 17:51:27.692341 storage/engine/rocksdb.go:149  closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.03s)
=== RUN   TestBatchGet
I160324 17:51:27.722508 storage/engine/rocksdb.go:149  closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.01s)
=== RUN   TestBatchMerge
I160324 17:51:27.729523 storage/engine/rocksdb.go:149  closing in-memory rocksdb instance
--- PASS: TestBatchMerge (0.01s)
=== RUN   TestBatchProto

```
Please assign, take a look and update the issue accordingly.

--- a/storage/scanner_test.go
+++ b/storage/scanner_test.go
@@ -305,15 +305,15 @@ func TestScannerEmptyRangeSet(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	ranges := newTestRangeSet(0, t)
 	q := &testQueue{}
-	s := newReplicaScanner(1*time.Millisecond, 0, ranges)
+	s := newReplicaScanner(time.Hour, 0, ranges)
 	s.AddQueues(q)
 	mc := hlc.NewManualClock(0)
 	clock := hlc.NewClock(mc.UnixNano)
 	stopper := stop.NewStopper()
 	defer stopper.Stop()
 	s.Start(clock, stopper)
-	time.Sleep(3 * time.Millisecond)
-	if count := s.Count(); count > 3 {
-		t.Errorf("expected three loops; got %d", count)
+	time.Sleep(time.Millisecond) // give it some time to (not) busy loop
+	if count := s.Count(); count > 1 {
+		t.Errorf("expected at most one loop, but got %d", count)
 	}
 }

